back/src/index.ts

```diff
@@ -13,6 +15,8 @@ app.get('/', (req, res) => {
   res.send('Hello World!');
 });
 
+app.use('/users', UserRouter);
```

Comment:
You mentioned "comments" last time for routes, is it interesting to add it now ?

Reply:
for the moment no because this route ll change in next week